src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class WindowAndCursorProps
+{
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public static void WindowWidth()
+    {
+        Assert.Throws<PlatformNotSupportedException>(() => Console.WindowWidth = 100);
+
+        // Validate that Console.WindowWidth returns some value in a non-redirected o/p.
+        RunInNonRedirectedOutput((data) => Console.WriteLine(Console.WindowWidth));
```

Comment:
I'm a little confused about this RunInNonRedirectedOutput due to the comment below citing that the CI system makes it difficult.  Is it just that this usage of it isn't actually problematic but it's also not really testing what it's designed to test?